test-support/helpers/ember-power-select.js

```diff
@@ -67,7 +67,7 @@ export default function() {
   const isEmberOne = Ember.VERSION.match(/1\.13/);
 
   Ember.Test.registerAsyncHelper('selectChoose', function(app, cssPath, value) {
-    const id = find(cssPath).find('.ember-power-select-trigger').attr('id').match(/ember-power-select-trigger-ember(\d+)/)[1];
+    const [,id] = find(cssPath).find('.ember-power-select-trigger').attr('id').match(/ember-power-select-trigger-ember(\d+)/);
```

Comment:
Maybe this kind of destructuring is the one that compiles to someting that uses `Symbol.iterator`?